plugins/publisher.proto

```diff
@@ -55,9 +55,18 @@ service Publisher {
     rpc CompleteOAuthFlow (CompleteOAuthFlowRequest) returns (CompleteOAuthFlowResponse){        
     }
 
+    // Configures a write back which can be used to write back a schema to the source.
+    // This must be implemented if the plugin manifest has `canConfigureWrite` set to true.
```

Comment:
I think we should have either `canConfigureWrite` and `canWrite` or else `canConfigureWriteBack` and `canWriteBack`, not a mix of `Write` and `WriteBack`.

We should also update the Schema message to add a DataflowDirection enum indicating whether it's a Read schema or a Write schema. That will let plugins with pre-defined write schemas return them in the DiscoverSchemasResponse.

The enum should set the Read value to 0, so that existing schemas will automatically be flagged as Read.